images/cass-config-builder/README.md

```diff
@@ -0,0 +1,29 @@
+<!--monopod:start-->
+# cass-config-builder
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/cass-config-builder` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cass-config-builder/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Minimal [cass-config-builder](https://github.com/datastax/cass-config-builder) container image.
+<!--overview:end-->
+
+<!--getting:start-->
+## Get It!
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/cass-config-builder:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+<!--body:end-->
```

Comment:
No docs on how to use this image - we'd expect the usual here, i.e links to docs / helm chart with example, docker (don't think docker is applicable here). 

We've done this in the test but we should have some notes here on how to install the chart, where to find it, and example to run it (with arf to override our images)